pipeline/Makefile

```diff
@@ -13,6 +13,7 @@ build/conda-env: environment.yml
 	mkdir -p build && touch $@
 
 build/batch: $(BATCH_PY_FILES) build/conda-env
+	conda env update -n hail-pipeline --file ../batch/environment.yml
```

Comment:
We needed to get the dependencies from the batch environment to pipeline. Here's a stack overflow post discussing how to do this: https://stackoverflow.com/questions/48158137/conda-importing-one-environment-yml-into-another. I chose the easiest option for now.